app/controllers/admin/downloads_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::DownloadsController < Admin::BaseAdminController
```

Comment:
There are other concepts of "download" in the admin site, for example the DQT file. Do you think we should be more specific in the naming of the class?

Reply:
Yeah, I think that makes sense, I do remember wondering if it could be namespaced into `PayrollRun` at one point, but I had a lot to do so parked it, but I like the idea it is:

`Admin::PayrollRun::DownloadController`

If I make it `PayrollRunDownloadController` do you think that is good enough?

Reply:
Yeah, I think `Admin::PayrollRunDownloadsController` would be alright.